.pylintrc

```diff
@@ -0,0 +1,5 @@
+[MESSAGES CONTROL]
```

Comment:
In order to get pylint to stop giving false negatives, I modified .pylintrc concurrently with pylint.yml. Why both and not one?